src/py/fly_by_features.py

```diff
@@ -191,10 +201,10 @@ def main(args):
 		model = tf.keras.models.load_model(args.model, custom_objects={'tf': tf})
 		model.summary()
 
-	flyby = FlyByGenerator(sphere, args.resolution, visualize=args.visualize, use_z=args.use_z, split_z=args.split_z)
+	flyby = FlyByGenerator(sphere, args.resolution, visualize=args.visualize, use_z=args.use_z, split_z=args.split_z, rescale_features=args.rescale_features)
 	
 	if args.point_features or args.out_point_id:
-		flyby_features = FlyByGenerator(sphere, args.resolution, visualize=args.visualize)
+		flyby_features = FlyByGenerator(sphere, args.resolution, visualize=args.visualize, rescale_features=args.rescale_features)
```

Comment:
We don't need the rescale features here. 
This part will generate the point id map. The point id map is encoded in RGB components and we should never rescale those. This will actually break the code when is extracting the XYZ coordinates. If you are generating samples, the XYZ coordinates are likely wrong.